### ISSUE TITLE
Basic recently processed file list

### ISSUE THREAD

**Issue by gsxgmpkt9020** [mifi/lossless-cut#2201]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

Basically, we always need to check what we worked recently and fetch it quickly. I think this is a basic function what a basic software need. (Especially when we select "dir unset" and always new files will keep old modified time).

I know I can check this from the developer tools console, but it is also need a clean and simple list for newbie.

For example, a finished files list for quickly open the folder the file processed. (Just like Internet browser downloaded file list)

![image](https://github.com/user-attachments/assets/fa85768e-a1f5-4a2c-a82d-b971a69c549f)
